cla_public/templates/privacy.html

```diff
@@ -622,7 +622,7 @@ <h2>{{ _('Complaints') }}</h2>
         <span class="locality">Wilmslow</span>
         <span class="locality">Cheshire</span>
```

Comment:
Just an observation: English counties almost always have different Welsh names.  In this case **Swydd Gaer**.  Do we want to bother with that?

Reply:
@MalcolmVonMoJ - I ran this text through translation and the translators opted to keep it in English, maybe the Post Office would have trouble sending post??